packages/blade/src/components/Collapsible/_decisions/decisions.md

```diff
@@ -0,0 +1,186 @@
+# Collapsible
+
+Collapsible is used to allow users to toggle the visibility of hidden content within a container.
+
+This document outlines the API of `Collapsible` component. This shares similarities with [Accordion's API](https://github.com/razorpay/blade/pull/1189).
+
+<div align="center">
+  <img src="./collapsible-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Collapsible](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-629874&t=sVxH3DOnx3L3F9rO-0)
+
+## Interactions
+
+Collapsible shares the opening / closing interaction with Accordion:
+
+### Accordion open
+
+https://github.com/razorpay/blade/assets/6682655/1a02d680-1832-4a70-a0a3-1a1f56f2b8b9
```

Comment:
lets put a motion of collapsible specifically?

Reply:
Don't have it on Figma, will put a screen recording.